src/SALib/analyze/delta.py

```diff
@@ -50,6 +53,10 @@ def analyze(
         The confidence interval level (default 0.95)
     print_to_console : bool
         Print results directly to console (default False)
+    Nr: int

```

Comment:
Once we settle on the API, this needs a space in between.
```suggestion
    Nr : int

```